assignment.js

```diff
@@ -65,7 +69,15 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
-
+const numbers = [3, 48, 29, 12, 18]
+let largestNumber = 0
+
+for(t = 0; t < numbers.length; t++) {
```

Comment:
@studiosok  nice work! I like the use of the`currentLargestNumber` variable, array indexing and a conditional to keep track of the current largest number. 

If you're interested I added an alternative approach (which violated the requirements by not using `currentLargestNumber` variable) that uses`sort()` method - see [more details here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L86)